web-bluetooth/read-characteristic-value-changed.js

```diff
@@ -0,0 +1,73 @@
+var batteryLevelCharacteristic;
+
+function onReadBatteryLevelButtonClick() {
+  log('Requesting Bluetooth Device...');
+  navigator.bluetooth.requestDevice(
+    {filters: anyDevice(), optionalServices: ['battery_service']})
+  .then(device => {
+    log('Connecting to GATT Server...');
+    return device.gatt.connect();
+  })
+  .then(server => {
+    log('Getting Battery Service...');
+    return server.getPrimaryService('battery_service');
+  })
+  .then(service => {
+    log('Getting Battery Level Characteristic...');
+    return service.getCharacteristic('battery_level');
+  })
+  .then(characteristic => {
+    log('Reading Battery Level...');
+    batteryLevelCharacteristic = characteristic;
+    batteryLevelCharacteristic.addEventListener('characteristicvaluechanged',
+        handleBatteryLevelChanged);
+    return batteryLevelCharacteristic.readValue();
+  })
+  .then(_ => {
+    document.querySelector('#startNotifications').disabled = false;
+    document.querySelector('#stopNotifications').disabled = false;
```

Comment:
If we haven't started notifications for a characteristic then we should probably disable the 'Stop Notifications' button. Similarly if we've started notifications then we should probably disable the 'Start Notifications". Would that make sense for this sample?